examples/c/Makefile

```diff
@@ -58,8 +59,9 @@ $(LIBBPF_OBJ): $(wildcard $(LIBBPF_SRC)/*.[ch] $(LIBBPF_SRC)/Makefile) | $(OUTPU
 		    install
 
 # Build BPF code
-$(OUTPUT)/%.bpf.o: %.bpf.c $(LIBBPF_OBJ) $(wildcard %.h) vmlinux.h | $(OUTPUT)
+$(OUTPUT)/%.bpf.o: %.bpf.c $(LIBBPF_OBJ) $(wildcard %.h) $(VMLINUX) | $(OUTPUT)
```

Comment:
hm... did you try adding `$(dir $@)` or `$(dir $(OUTPUT)/%)`, just curious. It would be nice to have a nice MKDIR message, similarly to $(OUTPUT) and $(OUTPUT)/libbpf cases

Reply:
Done